script/hassfest/docker.py

```diff
@@ -28,21 +28,8 @@
 
 # Home Assistant S6-Overlay
 COPY rootfs /
-
-# Needs to be redefined inside the FROM statement to be set for RUN commands
-ARG BUILD_ARCH
 # Get go2rtc binary
-RUN \
-    case "${{BUILD_ARCH}}" in \
-        "aarch64") go2rtc_suffix='arm64' ;; \
-        "armhf") go2rtc_suffix='armv6' ;; \
-        "armv7") go2rtc_suffix='arm' ;; \
-        *) go2rtc_suffix=${{BUILD_ARCH}} ;; \
-    esac \
-    && curl -L https://github.com/AlexxIT/go2rtc/releases/download/v{go2rtc}/go2rtc_linux_${{go2rtc_suffix}} --output /bin/go2rtc \
-    && chmod +x /bin/go2rtc \
-    # Verify go2rtc can be executed
-    && go2rtc --version
```

Comment:
should we still do the run check?

Reply:
I only added it in the first place to verify that `chmod +x` was successful. Not sure if it's worth to add an additional run statement for it